src/bosh-softlayer-cpi/softlayer/common/vm_utils.go

```diff
@@ -249,3 +229,31 @@ func GetLocalIPAddressOfGivenInterface(networkInterface string) (string, error)
 const ETC_HOSTS_TEMPLATE = `127.0.0.1 localhost
 {{.}}
 `
+
+// private methods
```

Comment:
These are functions, not methods.

Also, what happens when things don't exist or error?

I expected these functions to return errors so that error paths can be tested... unless you are testing elsewhere?

Finally, I am OK if you make these functions public so you can test them directly.